library/src/containers/Channels/NewOperations.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+
+import { OperationComponent } from './NewOperation';
+import { Toggle } from '../../components';
+
+import { bemClasses } from '../../helpers';
+import { useSpec } from '../../store';
+import { PayloadType } from '../../types';
+import { CONTAINER_LABELS, CHANNELS_TEXT } from '../../constants';
+
+export const OperationsComponent: React.FunctionComponent = () => {
+  const channels = useSpec().channels();
+
+  if (!Object.keys(channels).length) {
+    return null;
+  }
+
+  const className = CONTAINER_LABELS.CHANNELS;
+  const header = <h2>{CHANNELS_TEXT}</h2>;
+
+  const operationsList: React.ReactNodeArray = [];
+  Object.entries(channels).forEach(([channelName, channel]) => {
+    if (channel.hasPublish()) {
+      operationsList.push(
+        <li key={channelName}>
+          <OperationComponent
+            operationType={PayloadType.PUBLISH}
+            operation={channel.publish()}
+            channelName={channelName}
+            channel={channel}
+          />
+        </li>,
+      );
+    }
+    if (channel.hasSubscribe()) {
+      operationsList.push(
+        <li key={channelName}>
+          <OperationComponent
+            operationType={PayloadType.SUBSCRIBE}
+            operation={channel.subscribe()}
+            channelName={channelName}
+            channel={channel}
+          />
+        </li>,
+      );
+    }
+  });
+
+  return (
+    <section
+      className={bemClasses.element(className)}
+      id={bemClasses.identifier([className])}
```

Comment:
Why are you still using BEM in the new component? It should be using Tailwind instead.

Reply:
I know, but section only wraps content like this:

![image](https://user-images.githubusercontent.com/20404945/112468328-68d77d00-8d68-11eb-9674-bae1fe9d3975.png)

I wanna make styles here asyncapi/shape-up-process#86

Reply:
Oh ok! I'll take care of this task if you don't mind.